apps/web/src/server/applications/create/case/applications-create-case.controller.js

```diff
@@ -44,7 +44,7 @@ import {
 export async function viewApplicationsCreateCaseName(req, response) {
 	const { applicationId } = response.locals || {};
 	const { title, description } = applicationId
-		? await getApplicationDraft(applicationId)
+		? await getApplicationDraft(applicationId, '{"name":true, "description":true}')
```

Comment:
A suggestion around how these strings are generated to ensure the syntax would be to write `JSON.stringify({name: true, description: true})`

Reply:
another strategy could be just passing the keys as an array, something like `['name', 'description', 'applicants']` and let the `getApplicationDraft` service do the job of creating the query string

Reply:
@DavidJWhiteDev

Reply:
I like the idea of passing a proper array, and converting it.  I have seen that in the code elsewhere.  would look much cleaner!

Reply:
so if have an array of, say, `filters`, it could be something like:
```
const filterObject = {}
for(const filter of filters) {
filterObject = {...filterObject, [filter]: true}
}
const query = JSON.stringify(filterObject)
```